lib/elixir/lib/exception.ex

```diff
@@ -240,7 +240,7 @@ defmodule Exception do
   defp format_exit_reason(:noconnection), do: "no connection"
 
   defp format_exit_reason(:noproc) do
-    "no process"
+    "no process: the process is dead or no process is associated with the given name, possibly because its application isn't started"
```

Comment:
Should we go for "is not alive" instead of "is dead" because the later could be confused with process exiting during a call? Whereas `:noproc` is when the process did not exist.